.github/workflows/lint.yml

```diff
@@ -19,10 +19,15 @@ jobs:
       - uses: actions/checkout@v4
       - uses: pnpm/action-setup@v4
         with:
-          run_install: true
+          run_install: false
+      - uses: actions/setup-node@v4
+        with:
+          node-version: '20'
+          cache: 'pnpm'
```

Comment:
`actions/setup-node@v4` is called twice here, also I think we can keep `run_install` to true to avoid putting it manually afterwards?

Reply:
run_install was turned to false to prevent the warning of node version 18 being used while `pnpm install`. How is `actions/setup-node@v4` being called twice?

Reply:
See example here: https://github.com/pnpm/action-setup?tab=readme-ov-file#use-cache-to-reduce-installation-time

Reply:
nvm